### PR TITLE
7.1.x | LRDOCS-6391 Indexing framework minor updates

### PR DESCRIPTION
**Pull request by lipusz**
https://issues.liferay.com/browse/LRDOCS-6391

Hey Russ,

I've added some extra info from the internal skill map.

Cheers,
Tibor

### CODE REVIEW

File: develop/tutorials/articles/160-search/03-indexers.markdown [hunk 1]
```diff
@@ -67,7 +67,13 @@ For example, the
 [`GroupedModelDocumentContributor`](https://github.com/liferay/liferay-portal/blob/7.1.1-ga2/modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/contributor/document/GroupedModelDocumentContributor.java)
 contributes the `GROUP_ID` and `SCOPE_GROUP_ID` fields for all documents with a
 backing entity that's also a `GroupedModel`.
- 
+
```
Comment (rbohl) [inline develop/tutorials/articles/160-search/03-indexers.markdown]: Doesn't this duplicate what's directly above? :point_up:

Comment (rbohl) [reply develop/tutorials/articles/160-search/03-indexers.markdown]: See https://dev.liferay.com/en/develop/tutorials/-/knowledge_base/7-1/indexing-framework#indexing, the last paragraph.

Comment (lipusz) [reply develop/tutorials/articles/160-search/03-indexers.markdown]: Yeah, though it's better to explain what "type" means. It sounds vague to me.

Comment (rbohl) [reply develop/tutorials/articles/160-search/03-indexers.markdown]: Well I was actually referring to the type parameter in the class declaration, so I'll add the part about the component property, too

**Comment by rbohl** [comment]
superceded: https://github.com/lipusz/liferay-docs/pull/28

**Pull request closed without merging**